docs/starters.yml

```diff
@@ -1634,3 +1634,20 @@
     - GSAP ready
     - Nice devtools
     - Decorators
+- url: https://gatsby-sseon-starter.netlify.com/
+  repo: https://github.com/SeonHyungJo/gatsby-sseon-starter
+  description: Simple starter template for Gatsby
+  tags:
+    - Simple
+    - Blog
+    - Disqus
+    - Portfolio
```

Comment:
Looks like your starter is a blog but not a portfolio so you should remove this tag
```suggestion
```